src/home/secondary-features/image.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Box, Grid, makeStyles } from '@material-ui/core'
+
+import Source from 'home/secondary-features/image.svg'
+
+const useStyles = makeStyles(theme => ({
+  image: {
+    width: 480,
+
+    [theme.breakpoints.between(1, 350)]: { width: 190 },
+    [theme.breakpoints.between(350, 500)]: { width: 330 },
```

Comment:
Maybe instead of using fixed width, we could make the image responsive with:
```suggestion
    width: '100%',
    height: 'auto',
```

Reply:
It won't fix the whole problem unfortunately. This is how would it look on an iPad device, for example.
![image](https://user-images.githubusercontent.com/39530866/123602722-b816c900-d801-11eb-882e-30330fb7e867.png)